en/08_Changelogs/5.4.0.md

```diff
@@ -32,6 +33,25 @@ We have provided a severity rating of the vulnerabilities below based on the CVS
 
 ## Features and enhancements
 
+### Logged warning if allowed hosts have not been configured {#allowed-hosts-warning}
+
+If your site does not have one of the following configured, then a warning will now be logged on every request:
+
+- `SS_ALLOWED_HOSTS` environment variable
+- [`AllowedHostsMiddleware.AllowedHosts`](api:SilverStripe\Control\Middleware\AllowedHostsMiddleware->AllowedHosts) property
+
+If [`Director.alternate_base_url`](api:SilverStripe\Control\Director->alternate_base_url) is not set, the "host" header is used by Silverstripe CMS to determine what the host name is for your project. This is useful when creating absolute URLs, e.g. for use in emails.
+
+Notably this is used in [`Director::host()`](api:SilverStripe\Control\Director::host()), which in turn is called by many methods including [`Director::hostName()`](api:SilverStripe\Control\Director::hostName()), [`Director::protocolAndHost()`](api:SilverStripe\Control\Director::protocolAndHost()), [`Director::is_site_url()`](api:SilverStripe\Control\Director::is_site_url()), and [`Director::absoluteURL()`](api:SilverStripe\Control\Director::absoluteURL()).
```

Comment:
```suggestion
If [`Director.alternate_base_url`](api:SilverStripe\Control\Director->alternate_base_url) is set then it is often used instead instead of the "host" HTTP request header for determining the URL of the website. However there may be some scenarios where the host header is still used, so having `alternate_base_url` configured by itself will not prevent a warning be logged if allowed hosts are not configured.
```

Maybe this?

Reply:
Based on https://github.com/silverstripe/developer-docs/pull/692#discussion_r1955319621 it sounds like referencing `alternate_base_url` was confusing, so I've removed that and left the rest.

I.e. it doesn't matter that sometimes some methods won't use the host header if `alternate_base_url` is set - but it is useful to know some of the common methods that _do_ tend to use the host header.